### ISSUE TITLE
alpha_loss error

### ISSUE THREAD

**Issue by galefinch** [alirezakazemipour/Discrete-SAC-PyTorch#1]
hello, in Discrete-SAC-PyTorch/Brain/agent.py line 111
the alpha loss should be 
`alpha_loss = -(self.log_alpha.exp() * (log_probs.detach() + self.entropy_target)).mean()`
but the code is 
`alpha_loss = -(self.log_alpha * (log_probs.detach() + self.entropy_target)).mean()`
![image](https://user-images.githubusercontent.com/29545256/147736913-9f5e5c8f-c5e0-4603-be8c-d649be0f8bbe.png)

**Comment by akazemipour** [comment]
Hi,
Please read the following:
https://github.com/ku2482/sac-discrete.pytorch/issues/3#issuecomment-561274577

**Comment by galefinch** [comment]
Thank you!
